src/components/Body.js

```diff
@@ -19,13 +24,24 @@ export default function Body() {
     );
   }
 
-  const shortNews = news.articles.slice(0, 4);
+  const numberOfPages = Math.ceil(news.totalResults / articlesPerPage);
 
   return (
     <div className={styles["body"]}>
-      {shortNews.map((article, index) => (
+      {news.articles.map((article, index) => (
         <Articles key={index} article={article} />
```

Comment:
Change the name to Article because this component represent a single article and not the all of them